.github/workflows/cd.yml

```diff
@@ -0,0 +1,32 @@
+name: CD 
+
+on: 
+  push:
+    branches:
+      - main
+  
+jobs:
+  deploy_to_server:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and push
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          file: ./Dockerfile
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/roulette:latest
+      - name: Deploy to Server
+        run: |
+          sudo apt-get update && sudo apt-get install -y ansible
+          ansible-playbook -i inventory.ini -u root deploy.yml
```

Comment:
@aalysher could you please separate the steps with a newline.

Reply:
done